components/admin_console/admin_definition_plugins.jsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export let cache = null;
```

Comment:
does this need to be exported? As there are setting/getter functions for it.

I wonder if the `cache` has to be in the module, can it live in redux store? As with @marianunez 's proposed changed, would be good to store there, that way it'll be easier to test. Thoughts?